src/pages/zoeken/[slug]/[ie]/[name]/index.tsx

```diff
@@ -341,6 +341,7 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, description,
 	const showKeyUserPill = mediaInfo?.accessThrough?.includes(IeObjectAccessThrough.SECTOR);
 	const showVisitButton =
 		visitorSpace?.status === VisitorSpaceStatus.Active && canRequestAccess && !isKiosk;
+	const hasContentLicense = mediaInfo?.licenses?.includes(IeObjectLicense.BEZOEKERTOOL_CONTENT);
```

Comment:
This check is already part of the existing logic:
```
	const canRequestAccess =
		isNil(
			accessibleVisitorSpaces?.find((space) => space.maintainerId === mediaInfo?.maintainerId)
		) &&
		mediaInfo?.licenses?.includes(IeObjectLicense.BEZOEKERTOOL_CONTENT) &&
		isNil(mediaInfo.thumbnailUrl);
```

The issue is that someone can have access to a limited visitor space. So this check is too strict:
```
isNil(
	accessibleVisitorSpaces?.find((space) => space.maintainerId === mediaInfo?.maintainerId)
)
```

I think we can just omit it, since we already check that there is not thumbnail, so if there is not thumbnail, you don't have access to the content, but the object has content that is accessible.
So this check should do it:
```
	const showVisitButton =
		isNil(mediaInfo.thumbnailUrl) &&
		mediaInfo?.licenses?.includes(IeObjectLicense.BEZOEKERTOOL_CONTENT) &&
		visitorSpace?.status === VisitorSpaceStatus.Active &&
		!isKiosk;		
```
then we can remove the canRequestAccess boolean if it is not used elsewhere.

The current implementation is not correct since it will also show the button for visitor spaces that have an inactive or pending status.